docs/reference/eql/functions.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[eql-function-ref]]
+== EQL function reference
+++++
+<titleabbrev>Function reference</titleabbrev>
+++++
+
+experimental::[]
+
+{es} supports the following EQL functions:
+
+* <<eql-substring-fn>>
```

Comment:
Separate question should `fn` be a prefix instead of a suffix to make it easy to categorize, e.g. `eql-fn-substring`

Reply:
Good call. This'll help w/ copy/paste in the future. Moved to prefix with f03b9e11607.